content/docs/run-api-server/installing.mdx

```diff
@@ -5,25 +5,115 @@ order: 20
 
 import { CodeExample } from "components/CodeExample";
 
-To install Horizon, you have a choice: you can download a [prebuilt release for your target architecture](https://github.com/stellar/go/releases) and operation system or you can [build Horizon yourself](#Building). When either approach is complete, you will find yourself with a directory containing a file named `horizon`. This file is a native binary.
 
-After building or unpacking Horizon, you simply need to copy the native binary into a directory that is part of your PATH. Most unix-like systems have `/usr/local/bin` in PATH by default, so unless you have a preference or know better, we recommend you copy the binary there.
+## Installation Methods
 
-To test the installation, simply run `horizon --help` from a terminal. If the help for Horizon is displayed, your installation was successful. Note: some shells, such as zsh, cache PATH lookups. You may need to clear your cache (by using `rehash` in zsh, for example) before trying to run `horizon --help`.
+To install Horizon, you have a few choices. You can...
+  - install prebuilt binaries [from our repositories](#package-manager) via your package manager if running a Debian-based system,
+  - download a [prebuilt release](https://github.com/stellar/go/releases) for your target architecture and operation system, or
+  - [build Horizon yourself](#building) from scratch. 
 
-## Building
+The first method is recommended: not only do you ensure OS compatibility and dependency management, you'll also install some convenient wrappers that make running Horizon and Stellar-Core much simpler.
 
+### Package Manager
+SDF publishes new releases to its custom Ubuntu repositories. Follow [this guide](https://github.com/stellar/packages/blob/master/docs/adding-the-sdf-stable-repository-to-your-system.md#adding-the-sdf-stable-repository-to-your-system) to add the stable SDF repository to your system. Then, 
+
+<CodeExample>
+
+```bash
+sudo apt update && sudo apt install stellar-captive-core stellar-horizon
+```
+
+</CodeExample>
+
+[This page](https://github.com/stellar/packages/blob/master/docs/installing-individual-packages.md#installing-individual-packages) outlines the various commands that these packages make available.
+
+### Building
 Should you decide not to use one of our prebuilt releases, you may instead build Horizon from source. To do so, you need to install some developer tools:
 
-- A unix-like operating system with the common core commands (cp, tar, mkdir, bash, etc.)
+- A Unix-like operating system with the common core commands (cp, tar, mkdir, bash, etc.)
 - A compatible distribution of Go (Go 1.13 or later)
```

Comment:
Just want to verify that `Go 1.13 or later` is still accurate.

Reply:
Reviewed w/ team and updated accordingly in e7d8d0d9399533c10a83f087d9057eb9f1aaa7c0, good catch!